### DISCUSSION TITLE
changing/translating docker for desktop language. Windows

### DISCUSSION THREAD

**Discussion by azurejoga** [docker/roadmap#564] [Q&A]
Hello everyone, and greetings especially to the docker desktop developers!

Your app is excellent! thank you for that!

* Q
Is there any way to translate docker into other languages?
I see that it currently only supports English, how can I contribute to translations or change the language?

* Q
How can I even make a new PR to contribute and if this is available in your plans, add support for other languages.

**Comment by codenamehaven** [comment]
Hi, Just a humble community member here. As of the latest update from the Docker team in the roadmap issues, this feature has not yet been considered. Therefore, it has no near term plans at the moment. The feature having no part in the current roadmap, you will have to wait until it's being added by the team in the future. To stay updated and be notified, you can subscribe to the related issues.

**Comment by toyro967** [comment]
איני גם אשמח ונראה לי שבעידן הAI זה לא כל כך מסובך להתאים את האפליקציה לתמיכה בשפות מרובות
